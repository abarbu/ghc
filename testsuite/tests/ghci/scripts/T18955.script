:set -v1
:set -fbyte-code
:l T18955
