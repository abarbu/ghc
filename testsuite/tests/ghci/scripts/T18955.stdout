[1 of 1] Compiling Main             ( T18955.hs, interpreted )
Ok, one module loaded.
